warrior/Actions/CiRegressionActions/ci_regression_actions.py

```diff
@@ -329,3 +329,125 @@ def delete_tmp_dir(self):
+        the type specified in the type_ prefix
+        :Argument:
+            1. str_value - expected to be string
+            2. int_value - expected to be int
+            3. float_value - expected to be float
+            4. bool_value - expected to be bool
+            5. list_value - expected to be list
+            6. tuple_value - expected to be tuple
+            7. dict_value - expected to be dict
+            8. file_value - expected to be file
+        """
+        file_contents = "Checking file datatype in wtags"
+        status = True
+        err_msg = "{} is not an {} value but of type {}"
+        if type(str_value) is not str:
+            # this block checks if str_value is string type
+            print_error(err_msg.format(str_value, "str", type(str_value)))
+            status = False
+        if type(int_value) is not int:
+            # this block checks if int_value is int type
+            print_error(err_msg.format(int_value, "int", type(int_value)))
+            status = False
+        if type(float_value) is not float:
+            # this block checks if float_value is float type
+            print_error(err_msg.format(float_value, "float", type(float_value)))
+            status = False
+        if type(bool_value) is not bool:
+            # this block checks if bool_value is bool type
+            print_error(err_msg.format(bool_value, "bool", type(bool_value)))
+            status = False
+        if type(list_value) is not list:
+            # this block checks if list_value is list type
+            print_error(err_msg.format(list_value, "list", type(list_value)))
+            status = False
+        if type(tuple_value) is not tuple:
+            # this block checks if tuple_value is tuple type
+            print_error(err_msg.format(tuple_value, "tuple", type(tuple_value)))
+            status = False
+        if type(dict_value) is not dict:
+            # this block checks if dict_value is dict type
+            print_error(err_msg.format(dict_value, "dict", type(dict_value)))
+            status = False
+        if type(file_value) is not file:
+            # this block checks if file_value is file type
+            print_error(err_msg.format(file_value, "file", type(file_value)))
+            status = False
+        else:
+            actual_contents = file_value.read().strip()
+            if actual_contents != file_contents:
+                # this block checks if the contents of file type variable is expected
+                print_error("contents of the file {} is <<{}>> which does not match expected"
+                            " <<{}>>".format(file_value, actual_contents, file_contents))
+                status = False
+        return status
+
+    def check_values_from_datafile(self, system_name, strvar, langs, states,
+                                   currencys, ramspace, configfile, intvar,
+                                   anotherfile):
+        """get the values from datafile
```

Comment:
This comment get hidden...copying to make a new one
This keyword description should be
Verify the datatype of the value read from the datafile using either the tag or wtag feature